hnn_core/parallel_backends.py

```diff
@@ -86,10 +86,11 @@ def _clone_and_simulate(self, net, trial_idx):
         from hnn_core.network_builder import NetworkBuilder
         from hnn_core.network_builder import _simulate_single_trial
 
-        if trial_idx != 0:
-            net.params['prng_*'] = trial_idx
+        for param_key in net.params['prng_*'].keys():
+            net.params[param_key] += trial_idx
 
         neuron_net = NetworkBuilder(net)
+        neuron_net.net.trial_idx = trial_idx
```

Comment:
is this equivalent to https://github.com/jonescompneurolab/hnn-core/pull/171/files#diff-ca5f146e027d9dc6ef29ff4db9c21ca2R68 ? I have a suspicion it is not. Can you check?

Reply:
You're right, I messed this up. Fixed in https://github.com/jonescompneurolab/hnn-core/pull/171/commits/b340fe0150c7b333d40a56f3e87ea0c0ea85004f.